tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/PrefetchRedeliveryCountOpenwireTest.java

```diff
@@ -176,4 +192,119 @@ public void testExclusiveConsumerOrderOnReconnectionLargePrefetch() throws Excep
          }
       }
    }
+
+   @Test(timeout = 600_000)
+   public void testExclusiveConsumerTransactionalBatchOnReconnectionLargePrefetch() throws Exception {
```

Comment:
testExclusiveConsumerTransactionalBatchOnReconnectionLargePrefetch passes even without the changes

Reply:
darn, good catch, I now see that the assertion is being caught! to spit out the paging data and the error is lost. Will fix. thanks!